mLRS/Common/hal/esp/esp-hal.h

```diff
@@ -96,6 +96,10 @@
 #include "tx-hal-radiomaster-bandit-micro-900-esp32.h"
 #endif
 
+#ifdef TX_ELRS_RADIOMASTER_BANDIT_900_ESP32
+#include "tx-hal-radiomaster-bandit-micro-900-esp32.h"
```

Comment:
just a thought, should we maybe rename to "...-bandit-series-900-...."

Reply:
Done